src-gui/src/renderer/components/pages/swap/swap/in_progress/ReceivedQuotePage.tsx

```diff
@@ -4,17 +4,19 @@ import {
 } from "store/hooks";
 import CircularProgressWithSubtitle, {
   LinearProgressWithSubtitle,
-} from "../../CircularProgressWithSubtitle";
+} from "../components/CircularProgressWithSubtitle";
+import CancelButton from "../CancelButton";
 
 export default function ReceivedQuotePage() {
   const syncProgress = useConservativeBitcoinSyncProgress();
+  let progress = null;
```

Comment:
Same here. If we don't render the CancelButton at this level we can skip this mutable variable and just return like we did previously.